Update repositories.json to add Listonomicon

ajaxxxxxxxx opened the pull request:
Adding Listonomicon https://raw.githubusercontent.com/ajaxxxxxxxx/Listonomicon/main/modlists.json

SovnSkyrim commented:
[xUnit.net 00:00:01.16]       Xunit.Sdk.TrueException: MachineUrl/ModlistRepository is not valid! Allowed Characters are: 0-9, A-Z, a-z, _ and - use https://regex101.com/r/cVYtyA/2to test: "Listonomicon/Listonomicon" ("Listonomicon")
 
 Your machine URL is not valid.

This is used internally to separate the list from others, id suggest putting "listonomicon" in there.

Your readme is very lackluster, you'll have a lot of complains (and so will we in the support discord); I'd heavily suggest checking how other lists have it setup.

The only blocking thing for the merge is the machineURL error for now.

ajaxxxxxxxx commented:
Thank you. I incorporated the feedback (see the link to our google doc of install instructions now in the readme) and replaced the machineURL with just "Listonomicon." Please let me know if this satisfies your concerns on the readme and fixes the issues with the pull request. Thanks!

SovnSkyrim commented:
Checks have passed now. Congratz.
